src/resources/fundingLogos/itau_vertical.svg

```diff
@@ -0,0 +1,34 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 350 105">
```

Comment:
Can we please just use a single SVG file, which works for both horizontal and vertical?